dupserver.go

```diff
@@ -88,10 +88,24 @@ func sendResultDup(client *client.Client, item *answer, dupAddress string, confi
 }
 
 func enqueueDupServerResult(config *configurationStruct, result *answer) {
+	duplicateResult := &answer{
```

Comment:
it might be enough to create a shallow copy like `duplicateResult := *item`. Since the struct only contains simple numbers and strings. But i haven't tried.

Reply:
Sure, I'll try that now